lib/main.js

```diff
@@ -0,0 +1,41 @@
+var converter = require('./converter'),
+    util = require('./util');
+
+module.exports = {
+    convert: function (options, callback) {
+        if (!options.output) {
+            callback(new Error('Output path must be specified.'));
+            return;
+        }
+        if (!options.input) {
+            callback(new Error('Input file path must be specified.'));
+            return;
+        }
+        if (!options.outputVersion) {
+            callback(new Error('Output version must be specified.'));
+            return;
+        }
+        if (!options.inputVersion) {
+            callback(new Error('Input version must be specified.'));
+            return;
+        }
+
+        util.loadJSON(options.input, function (err, collection) {
```

Comment:
Seems like you have assumed that loadJSON is default... note that we are also planning stdin input and stdout output if these are not provided. We can go ahead with this and keep a todo to get that coded before merging with develop.